knowledge_repo/app/utils/requests.py

```diff
@@ -22,15 +22,22 @@ def from_request_get_feed_params(request):
     :rtype: object
     """
 
-    feed_params = {}
-
-    feed_params["filters"] = request.args.get('filters')
-    feed_params["authors"] = request.args.get('authors')
-    feed_params["start"] = int(request.args.get('start', 0))
-    feed_params["results"] = int(request.args.get('results', 10))
-    feed_params["sort_by"] = inflection.underscore(
-        request.args.get('sort_by', 'updated_at'))
-    feed_params["sort_desc"] = not bool(request.args.get('sort_asc', ''))
+    feed_params = {'kr': None, 'filters': None, 'authors': None, 'start': 0, 'results': 10, 'sort_by': 'updated_at', 'sort_desc': True}
+    # default values of all filters if they are not present in url
+
+    qidx = url.find('?')
+    posts = []
+    if qidx != -1:
+      url_search = url[qidx+1:]
+      url_search_arr = url_search.split('&')
+      for filter in url_search_arr:
+        query, val = filter.split('=')
+        if '%2F' in val:
```

Comment:
whenever we use '/' in url for our use, it gets converted in '%2F' in requests.url, so explicitly converting it back to '/'

Reply:
yea that is what i'm asking. Is it causing an issue when it gets converted to %F? bcu that's typically how  URLs look like.

Reply:
Kr name contains a '/'. If it is left as '%2F', then it will cause problems (Because everywhere else it is assumed to have '/' between project id and kr name including database)

Reply:
In my experience it didn't. But then its good that you have done it anyway.